tf_agents/agents/ppo/examples/v2/train_eval.py

```diff
@@ -192,6 +215,23 @@ def train_eval(
         observers=[replay_buffer.add_batch] + train_metrics,
         num_episodes=collect_episodes_per_iteration)
 
+    # Initialize normalization parameters with random trajectories
+    for _ in range(norm_init_episodes):
+      init_replay_buffer = tf_uniform_replay_buffer.TFUniformReplayBuffer(
```

Comment:
No need to have the replay buffer recreated on every iteration. Also would it make more sense to do the agent normalizer initialization on more than one episode?